Project.toml

```
@@ -0,0 +1,22 @@
+name = "MetaImageFormat"
+uuid = "1950589f-4d68-56f0-9b94-9d8646217309"
+version = "0.2.1"
+
+[deps]
+AxisArrays = "39de3d68-74b9-583c-8d2d-e117c070f3a9"
+ColorVectorSpace = "c3611d14-8923-5661-9e6a-0046d554d3a4"
+Colors = "5ae59095-9a9b-59fe-a467-6f913c188581"
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+FixedPointNumbers = "53c48c17-4a7d-5ca2-90c5-79b7896eea93"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
+MappedArrays = "dbb5928d-eab1-5f90-85c2-b9b0edb7c900"
+
+[compat]
+julia = "≥ 1.0.0"
```

```suggestion
julia = "1"
```
It's safe to bet that most packages will be broken by Julia 2.0, so declaring that this only supports Julia 1 proactively ensures that the package system will install a version that is known to work.